src/http/api/account.ts

```diff
@@ -39,75 +43,37 @@ type FollowAccount = Pick<
 /**
  * Create a handler to handle a request for an account
  *
- * @param accountService Account service instance
+ * @param db Database client
```

Comment:
Are these param comments useful to you?

Reply:
not particularly no, just habit and i get a nice typescript description. Maybe we only include them when extra context is needed? (i.e `@param db Database client that can only read and not write`)